challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |string| string.capitalize }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select { |string| string == "dog" }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.select { |string| string != "dog" }
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.length
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input[0..1]
```

Comment:
I think `input.first(2)` works too. Using a range like you did here is great when you need more flexibility.